src/Connector.AzureDataLake/AzureDataLakeConstants.cs

```diff
@@ -74,8 +102,46 @@ public class AzureDataLakeConstants : ConfigurationConstantsBase, IAzureDataLake
                     displayName = DirectoryName,
                     type = "input",
                     isRequired = true
-                }
+                },
+                new Control
+                {
+                    name = OutputFormat,
+                    displayName = "JSON/Parquet/CSV",
+                    type = "input",
+                    isRequired = true,
+                },
```

Comment:
If a customer has a configured data lake export target and upgrade. This control needs to be set to JSON

Reply:
in the job data, it's set to json
`        public string OutputFormat => GetConfigurationValue(AzureDataLakeConstants.OutputFormat) as string ?? OutputFormats.Json;
`